timescaledb/how-to-guides/analytics/about-analytics.md

```diff
@@ -0,0 +1,75 @@
+# About Timescale Analytics
+Timescale Analytics is a PostreSQL extension containing a specialized set of
+functions that allow you to to analyze time-series data. You can use it to
+analyze anything you have stored as time-series data, including IoT devices, IT
+systems, marketing analytics, user behavior, financial metrics, and
+cryptocurrency. Timescale Analytics uses [Promscale][doc-promscale], an
+open-source analytics platform, for Prometheus monitoring data.
+
+Timescale Analytics allows you to perform critical time-series queries quickly,
+analyze time-series data, and extract meaningful information. It aims to
+identify, build, and combine all of the functionality SQL needs to perform
+time-series analysis into a single extension.
+
+## Tools for graphing
+Timescale Analytics brings graphing functions to the database. This allows you
+to choose your graphing front-end based on how well it does graphing, not on how
+well it does data analytics. It also allows you to run queries that stay
```

Comment:
We should probably revise these to link to the experimental features that correspond, again, as we write the new blog post, thsi should become more clear

Reply:
Yeah, I cut a lot out here, but thought this was still an important point. Happy to remove it, though, if it doesn't accurately describe things.